app/src/main/java/org/oppia/android/app/topic/conceptcard/ConceptCardFragmentPresenter.kt

```diff
@@ -52,6 +59,18 @@ class ConceptCardFragmentPresenter @Inject constructor(
       it.viewModel = viewModel
       it.lifecycleOwner = fragment
     }
+
+    viewModel.conceptCardLiveData.observe(
+      fragment,
+      Observer {
+        val explanation = htmlParserFactory
+          .create(resourceBucketName, entityType, skillId, /* imageCenterAlign= */true)
+          .parseOppiaHtml(it.explanation.html, view)
+
+        view.text = explanation
```

Comment:
```suggestion
        view.text = htmlParserFactory
          .create(resourceBucketName, entityType, skillId, /* imageCenterAlign= */ true)
          .parseOppiaHtml(it.explanation.html, view)
```

Small simplification & space fix.